src/taxonomy/export-modal/ExportModal.scss

```diff
@@ -0,0 +1,9 @@
+.taxonomy-export-modal {
+  .pgn__form-radio {
+    width: 100%;
+  }
+
+  .pgn__form-radio > div {
+    width: 100%;
+  }
+}
```

Comment:
Removing this bit doesn't seem to affect the UI much. Is it really needed?

Reply:
Yes, it's used to fix this:

>Could we extend the clickable area to include anything within the blue block in the screenshot above? Instinctively, I tried to click here in order to select the radio button
> ![image](https://github.com/openedx/frontend-app-course-authoring/assets/11827305/e56271fb-d69b-40bb-8565-cf90acedcb3b)

Reply:
Hmm, this will solve this particular issue, but I wonder if people are really going to click that far away from the label. 

Perhaps you can get a middle ground by adding a `labelClassName` of `pr-6` which will add some padding to the right of the label making that area clickable. 

I don't think this is a blocker in any way, but custom CSS overrides like this generally make theming harder.

Reply:
> Perhaps you can get a middle ground by adding a labelClassName of pr-6 which will add some padding to the right of the label making that area clickable.

I believe that the space created is very small and would not solve the issue.

![Captura desde 2023-12-18 10-32-47](https://github.com/openedx/frontend-app-course-authoring/assets/11827305/3e38d98c-345d-4cce-a34c-412f80d6ca83)

> Hmm, this will solve this particular issue, but I wonder if people are really going to click that far away from the label.

I think it's a UX decision. Until @ali-hugo  returns from his holidays, I can leave a comment in the code